Mage.Sets/src/mage/cards/a/Aberrant.java

```diff
@@ -41,7 +39,11 @@ public Aberrant(UUID ownerId, CardSetInfo setInfo) {
         this.addAbility(TrampleAbility.getInstance());
 
         // Heavy Power Hammer -- Whenever Aberrant deals combat damage to a player, destroy target artifact or enchantment that player controls.
-        this.addAbility(new AberrantTriggeredAbility());
+        Ability ability = new DealsCombatDamageToAPlayerTriggeredAbility(new DestroyTargetEffect(), false, true);
+        ability.withFlavorWord("Heavy Power Hammer");
+        ability.addTarget(new TargetPermanent(filter));
+        ability.setTargetAdjuster(new DamagedPlayerControlsTargetAdjuster());
```

Comment:
Just for idea -- looks like adding "target adjuster with blueprint target" can be a is standard code style. Also it can simplify target filter check (see another comment about controller/owner runtime check) and make it verify check instead (wrong filter will be catches on card constructor instead runtime usage).

I'm not sure is it safe to keep card without any targets (before first call of target adjuster). If it safe then it can be used without `ability.addTarget`:
* `ability.setTargetAdjuster(new DamagedPlayerControlsTargetAdjuster(new TargetPermanent(filter)));`

If it unsafe and target required before real usage (current code do it) then can be added in `setTargetAdjuster(TargetAdjuster` method like:
* adjuster interface:
``` java
public interface TargetAdjuster extends Serializable {

    // Warning: This is not Copyable, do not use changeable data inside (only use static objects like Filter)
    void adjustTargets(Ability ability, Game game);

    /**
     * Add default blueprint target to the ability
     */
    default void addDefaultTargets(Ability ability) {
    }
}
```
* ability:
``` java
    @Override
    public AbilityImpl setTargetAdjuster(TargetAdjuster targetAdjuster) {
        this.targetAdjuster = targetAdjuster;
        this.targetAdjuster.addDefaultTargets(this);
        return this;
    }
```
* adjuster implementation
``` java 
    @Override
    public void addDefaultTargets(Ability ability) {
        ability.addTarget(this.blueprintTarget.copy());
    }
```

Reply:
> I'm not sure is it safe to keep card without any targets (before first call of target adjuster). If it safe then it can be used without `ability.addTarget`:

It is not safe, see #12102 which fixed the bug that implementation causes.